src/sentry/runner/initializer.py

```diff
@@ -539,3 +543,110 @@ def on_configure(config):
 
     if 'south' in settings.INSTALLED_APPS:
         skip_migration_if_applied(settings, 'social_auth', 'social_auth_association')
+
+
+def validate_snuba():
+    """
+    Make sure everything related to Snuba is in sync.
+
+    This covers a few cases:
+
+    * When you have features related to Snuba, you must also
+      have snuba fully configured correctly to continue.
+    * If you have Snuba specific search/tagstore/tsdb backends,
+      you must also have a snuba compatible eventstream backend
+      otherwise no data getting written into Snuba.
+    * If you only have Snuba related eventstream, yell that you
+      probably want the other backends otherwise things are weird.
+    """
+    if not settings.DEBUG:
+        return
+
+    # This is a gross hardcoded list, but there's no
+    # other sensible way to manage this right now without augmenting
+    # features themselves in the manager with detections like this.
+    snuba_features = (
```

Comment:
Can you keep this list in `features.py` so that if we add more we can remember to update them there?

Reply:
Sure, and import from there? Makes sense.

Reply:
Done. Good suggestion.